docker/dockerfile.merlin

```diff
@@ -11,6 +11,9 @@ FROM ${DLFW_IMAGE} as dlfw
 FROM ${BASE_IMAGE} as build
 
 # Args
+ARG TARGETOS
+ARG TARGETARCH
+
```

Comment:
If not specified （which can cause all kinds of issues）, automatically filled by `docker build`.